homeassistant/components/otp/sensor.py

```diff
@@ -76,6 +76,7 @@ class TOTPSensor(SensorEntity):
     def __init__(self, name: str, token: str) -> None:
         """Initialize the sensor."""
         self._attr_name = name
+        self._attr_unique_id = token
```

Comment:
Hmm... Could this token maybe be changed in the future using a reconfigure step?

Reply:
nope, that wouldn't make sense, maybe change the associated name, but not the token. In that case i would say to just create a new entry.